wallet-webapps-common/src/main/webapp/vue-app/components/SendTokensForm.vue

```diff
@@ -594,14 +596,20 @@ export default {
       }
       this.$refs.sendTokensForm.open();
       if (!this.isSameNetworkVersion){
-        this.$root.$emit('show-alert', {type: 'warning',message: `${this.$t('exoplatform.wallet.warn.networkVersion')}<br>${this.walletUtils.getNetworkLink()}`});
+        this.showAlert('warning',`${this.$t('exoplatform.wallet.warn.networkVersion')}\n${this.walletUtils.getNetworkLink()}`);
       }
       if (!this.isSameAddress){
-        this.$root.$emit('show-alert', {type: 'warning',message: this.$t('exoplatform.wallet.warn.selectedAddress')});
+        this.showAlert('warning',this.$t('exoplatform.wallet.warn.selectedAddress'));
```

Comment:
we should display a button to end user to switch the account, as discussed with @mzorai